data/dirt belt jobs.txt

```diff
@@ -0,0 +1,279 @@
+# Copyright (c) 2017 by Michael Zahniser
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+mission "Drought Relief"
+	name `Drought relief to <planet>`
+	job
+	repeat
+	description `The farming world of <destination> is currently experiencing a drought. Deliver <cargo> to the planet by <date> for <payment>.`
+	cargo "drought relief" 25 2 .05
+	deadline
+	to offer
+		random < 10
+	source
+		government "Republic"
+	destination
+		distance 3 10
+		attributes "dirt belt"
+		attributes "farming"
+	on complete
+		payment 10000 160
+		dialog "You drop off your cargo of <commodity> and collect your payment of <payment>. A group of <planet> citizens thank you and bid you farewell."
+
+
+mission "To Remembrance Day celebration [0]"
+	name `Remembrance Day celebration`
+	job
+	repeat
+	description `Bring <fare> to <destination> by <date> for the Remembrance Day celebration. Payment is <payment>.`
+	passengers 4 4 .5
+	to offer
+		random < 60
+		or
+			and
+				month == 12
+				day >= 21
+			month == 1
+			and
+				month == 2
+				day < 19
+	to fail
+		month == 2
+		day == 19
+	source
+		attributes "dirt belt"
+	destination
+		distance 3 15
+		attributes "dirt belt"
+	on complete
+		payment 1000 180
+		dialog "You wish your <passengers> the best of luck on <planet>. The children run off your ship in green face paint, and the parents pay you <payment>."
+
+
+mission "To Remembrance Day celebration [1]"
+	name `Remembrance Day celebration`
+	job
+	repeat
+	description `Deliver <cargo> to <destination> by <date> in preparation for the Remembrance Day celebration. Payment is <payment>.`
+	cargo random 5 2 .1
+	to offer
+		random < 60
+		or
+			and
+				month == 12
+				day >= 21
+			month == 1
+			and
+				month == 2
+				day < 19
+	to fail
+		month == 2
+		day == 19
+	source
+		attributes "dirt belt"
+	destination
+		distance 3 15
+		attributes "dirt belt"
+	on complete
+		payment 1000 180
+		dialog `Spaceport workers already festively dressed for Remembrance Day remove the cargo of <commodity> from your ship and hand you your payment of <payment>.`
+
+
+mission "From Remembrance Day celebration [0]"
+	name `Return from Remembrance Day`
+	job
+	repeat
+	description `Bring <fare> home to <destination> after the Remembrance Day celebration. Payment is <payment>.`
+	passengers 4 4 .5
+	to offer
+		random < 60
```

Comment:
malformed ConditionSet here, needs an `or` below.
Seems kinda high offer rate too, considering it offers for up to 8 weeks. It may be better to rewrite this so that the offer rate decreases as time goes on. You'd probably need features from #3128 to do that smoothly, but could use more randoms to be a bit more piecewise about it.